README.md

```diff
@@ -124,7 +124,7 @@ This will show the price of the Bonsai Tree as being reduced even more
 15. Run the following command to get the external IP address  of the first worker node in your cluster
 ```bash
    # Substitute your cluster name e.g. user05-cluster for [YOUR_CLUSTER_NAME]
```

Comment:
since [YOUR_CLUSTER_NAME] is not in the command you need something different here now. Replace with

`# If you don't have the $USERNAME environment variable in the command set, replace $USERNAME with your lab user id.`